src/Message/AbstractRequest.php

```diff
@@ -421,6 +421,27 @@ public function setProductReference($value)
         return $this->setParameter('productReference', $value);
     }
 
+    /**
+     * Returns the invoice reference, which is normally the first element of invoice references array
```

Comment:
i think `which is normally the first element of invoice references array` is more confusing than helpful. probably fine without it unless you disagree